static/js/containers/ProfileFormContainer.js

```diff
@@ -81,78 +81,18 @@ class ProfileFormContainer extends React.Component {
     }
     dispatch(updateProfile(username, profile));
     this.updateProfileValidation(this.props, profile, validator);
-  }
+  };
 
   setProfileStep: Function = (step: string): void => {
     const { dispatch } = this.props;
     dispatch(setProfileStep(step));
   };
```

Comment:
I left this as-is because we use `this.setProfileStep` on `ProfilePage`, but now that I think about it we could just use `createActionHelper` there too...